src/Model/Group.php

```diff
@@ -417,6 +417,7 @@ public static function sidebarWidget($every = 'contacts', $each = 'group', $edit
 			'grouppage' => 'group/',
 			'$edittext' => L10n::t('Edit group'),
 			'$ungrouped' => $every === 'contacts' ? L10n::t('Contacts not in any group') : '',
+			'$ungrouped_selected' => (($group_id === 'none') ? 'group-selected' : ''),
```

Comment:
What's the difference between `$group_id === ''` and `$group_id === 'none'`?

Reply:
I have pushed a fix for that, was a sloppy oversight. group_id is now none/everyone/<id>.